libraries/SoaringController/MatrixMath.cpp

```diff
@@ -0,0 +1,269 @@
+/*
+ *  MatrixMath.cpp Library for MatrixMath
+ *
+ *  Created by Charlie Matlack on 12/18/10.
+ *  Modified from code by RobH45345 on Arduino Forums, taken from unknown source.
+ *  MatrixMath.cpp 
+ */
+
+//#include "Arduino.h"
+#include "MatrixMath.h"
+
+#define NR_END 1
+
+MatrixMath::MatrixMath()
+{
+}
+
+// Matrix Printing Routine
+// Uses tabs to separate numbers under assumption printed float width won't cause problems
+/*void MatrixMath::MatrixPrint(float* A, int m, int n, char[] label){
+    // A = input matrix (m x n)
+    int i,j;
+    Serial.println();
+    Serial.println(label);
+    for (i=0; i<m; i++){
+        for (j=0;j<n;j++){
+            Serial.print(A[n*i+j],4);
+            Serial.print("\t");
+        }
+        Serial.println();
+    }
+}
+*/
+
+void MatrixMath::MatrixCopy(float* A, int n, int m, float* B)
+{
+    int i, j, k;
+    for (i=0;i<m;i++)
```

Comment:
Thanks! This is the kind of style guide I was looking for. I'll go through it and reformat the code appropriately.